src/assets/scripts/client/aircraft/ModeControl/modeControlConstants.js

```diff
@@ -0,0 +1,49 @@
+export const MCP_ALTITUDE_MODES = {
+    HOLD: 'HOLD',
+    APPROACH: 'APPROACH',
+    LEVEL_CHANGE: 'LEVEL_CHANGE',
+    OFF: 'OFF',
+    VERTICAL_SPEED: 'VERTICAL_SPEED',
+    VNAV: 'VNAV'
+};
+
+export const MCP_SPEED_MODES = {
+    LEVEL_CHANGE: 'LEVEL_CHANGE',
+    N1: 'N1',
+    OFF: 'OFF',
+    HOLD: 'HOLD',
+    VNAV: 'VNAV'
+};
+
+export const MCP_HEADING_MODES = {
+    HOLD: 'HOLD',
+    LNAV: 'LNAV',
+    OFF: 'OFF',
+    VOR_LOC: 'VOR_LOC'
+};
+
+export const MCP_AUTOPILOT_MODES = {
+    ON: 'ON',
+    OFF: 'OFF'
+};
```

Comment:
we can control that pretty easily with a class method.  meaning, when we turn autopilot on we do those checks.